devito/interfaces.py

```diff
@@ -89,7 +91,7 @@ class DenseData(SymbolicData):
     therefore do not support time derivatives. Use :class TimeData: for
     time-varying griad data.
     """
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, disk_path=None, **kwargs):
```

Comment:
`disk_path` will come in as part of `**kwargs`. Please pick it up from inside the dict and remove this from this line.